examples/llama_example.py

```diff
@@ -20,6 +20,29 @@
 model = open_gpt.create_model(
     'decapoda-research/llama-7b-hf', precision='fp16', device_map='balanced'
 )
+
+# llama + LoRA
+# model = open_gpt.create_model(
+#     'decapoda-research/llama-7b-hf',
+#     precision='bit8',
+#     adapter_name_or_path='jinaai/alpaca-lora',
+#     device_map='balanced',
+# )
+# llama + LoRA using local ckpt file
+# model = open_gpt.create_model(
+#     'decapoda-research/llama-7b-hf',
+#     precision='bit8',
+#     adapter_name_or_path='./my-lora',
+#     device_map='balanced',
+# )
+# llama + QLoRA using local ckpt file
+# model = open_gpt.create_model(
+#     'decapoda-research/llama-7b-hf',
+#     precision='bit4',
+#     adapter_name_or_path='./my-qlora',
+#     device_map='balanced',
```

Comment:
I would suggest you use a separate new example `llama_lora_example.py` to make the example more clear and easy to understand.